docs/application/dotnet/guides/nui/xaml/define-layout.md

```diff
@@ -0,0 +1,151 @@
+
+## Create XAML
+
+You can use XAML to easily implement the layout shown in the following figure:
+
+![AmbientPage](./media/AmbientPage.PNG)
+
+### Prerequisites
+
+Ensure that the following prerequisites are met:
+
+- `x:Class` specifies the namespace and the class name for a class defined in the XAML file. The class name must match the class name of the code-behind file.
+- The filename of the XAML file must be same as the class name.
+
+To implement the layout, you can use the following code:
+
+```xml
+<ContentPage x:Class="Tizen.NUI.Examples.AmbientPage"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+
+  <ImageView Name="photo1" ResourceUrl="*Resource*/picture_07.jpg" Position2D="47,52" Size2D="440,440" />
+  <ImageView Name="photo2" ResourceUrl="*Resource*/picture_08.jpg" Position2D="47,512" Size2D="440,520" />
+  <ImageView Name="photo3" ResourceUrl="*Resource*/picture_02.jpg" Position2D="970,50" Size2D="440,560" />
+  <ImageView Name="photo4" ResourceUrl="*Resource*/picture_03.jpg" Position2D="970,630" Size2D="440,400" />
+  <ImageView Name="photo5" ResourceUrl="*Resource*/picture_04.jpg" Position2D="510,50" Size2D="440,280" />
+  <ImageView Name="photo6" ResourceUrl="*Resource*/picture_05.jpg" Position2D="510,350" Size2D="440,340" />
+  <ImageView Name="photo7" ResourceUrl="*Resource*/picture_06.jpg" Position2D="510,710" Size2D="440,320" />
+  <ImageView Name="photo8" ResourceUrl="*Resource*/picture_01.jpg" Position2D="1431,671" Size2D="440,360" />
+  
+  <ImageView Name="photo00" ResourceUrl="*Resource*/bg_textbox_w.png" Position2D="1431,51" Size2D="440,600" />
+
+  <View Name="num_area" Size2D="440,160" Position2D="1431,51">
+    <ImageView Name="photo01" ResourceUrl="*Resource*/num_w_2.png" Position2D="0,0" Size2D="110,160" />
+    <ImageView Name="photo02" ResourceUrl="*Resource*/num_w_4.png" Position2D="110,0" Size2D="110,160" />
+    <ImageView Name="photo03" ResourceUrl="*Resource*/num_w_th.png" Position2D="220,0" Size2D="220,160" />
+  </View>
+
+  <View Name="mon_area" Size2D="440,194" Position2D="1431,211">
+    <ImageView Name="photo10" ResourceUrl="*Resource*/mon_w_12.png" Position2D="0,0" Size2D="440,194" />
+  </View>
+
+  <View Name="year_area" Size2D="440,210" Position2D="1431,425">
+    <ImageView Name="photo21" ResourceUrl="*Resource*/year_w_2.png" Position2D="0,0" Size2D="110,210" />
+    <ImageView Name="photo22" ResourceUrl="*Resource*/year_w_0.png" Position2D="110,0" Size2D="110,210" />
+    <ImageView Name="photo23" ResourceUrl="*Resource*/year_w_1.png" Position2D="220,0" Size2D="110,210" />
+    <ImageView Name="photo24" ResourceUrl="*Resource*/year_w_9.png" Position2D="330,0" Size2D="110,210" />
+  </View>
+
+</ContentPage>
+```
+
+> **Note**
+>
+> In this code, the relative path of the image (relative to the location of the exe file) is used. You can also use the absolute path of the image.
```

Comment:
@mijin85cho, thank you.
The reason I suggested usage of Note tag was because, according to me it was important for the user to know that they can also set the absolute path and not only the relative path.

I agree with your suggested rephrase.
@dongsug-song,
Request you to please go through the following comment and provide your opinion:

https://github.com/Samsung/tizen-docs/pull/1007#discussion_r372720551